config/routes.rb

```diff
@@ -10,6 +10,7 @@
 
   namespace :account do
     get '/dashboard', to: 'dashboard#index'
+    get '/top_users', to: 'dashboard#top_users'
```

Comment:
```suggestion
    get '/top-users', to: 'dashboard#top_users'
```
please use ```-``` for url instead ```_```